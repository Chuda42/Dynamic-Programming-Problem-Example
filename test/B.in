leer_datos
10 20 .
leer_rendimiento
11 1 .
max_datos_procesados
planificacion_optima
leer_datos
10 50 .
leer_rendimiento
10 1 .
max_datos_procesados
planificacion_optima
Fin
