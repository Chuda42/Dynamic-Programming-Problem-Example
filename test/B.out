1>2>3>Max Datos Procesados: 11
4>Cantidad Datos Planificacion: 11
5>6>7>Max Datos Procesados: 11
8>Cantidad Datos Planificacion: 11
9>Fin.
